src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -2036,6 +2075,33 @@ public static void LogRegisteredTypes()
         }
         
         /// <summary>
+        /// Loads the external srializers and places them into a hash set
+        /// </summary>
+        /// <param name="type">The list of types that implement <see cref="IExternalSerializer"/></param>
+        private static void RegisterSerializationProviders(IEnumerable<Type> providerTypes)
+        {
+            if (providerTypes == null)
+            {
+                return;
+            }
+
+            providerTypes.ToList().ForEach(
```

Comment:
There's no need to do `ToList()` here. Could simply do `foreach(var providerType in providerTypes)`, right?